### PR TITLE
Add table of contents to API

### PR DESCRIPTION
**Pull request by evansiroky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,7 +248,7 @@ Translates to coordinate string.
 ```js
 var lonlat = require('@conveyal/lonlat')
 
-var str = lonlat.toString({ lat: 12, long: 34 })   // '12,34'
+var str = lonlat.toString({ lat: 12, lng: 34 })   // '12,34'
```
Comment (trevorgerhardt) [inline README.md:251]: Always use `lon`

Comment (trevorgerhardt) [reply README.md:251]: Especially in the docs

**Review by trevorgerhardt — CHANGES_REQUESTED** [review]

**Comment by trevorgerhardt** [comment]
@evansiroky we really should be using a documentation generator for these libraries. This would all be automated.

**Comment by evansiroky** [comment]
Closing in favor of #10

**Pull request closed without merging**